WallsAndHoles/toolmanager.cpp

```diff
@@ -0,0 +1,37 @@
+#include "toolmanager.h"
```

Comment:
In general the tool system will likely evolve in a lot of ways as we figure the rest of the system out, mainly in terms of using QActions and such